src/croc/croc.go

```diff
@@ -646,7 +646,7 @@ func (c *Client) Send(filesInfo []FileInfo, emptyFoldersToTransfer []FileInfo, t
 	if c.Options.RelayPassword != models.DEFAULT_PASSPHRASE {
 		flags.WriteString("--pass " + c.Options.RelayPassword + " ")
 	}
-	fmt.Fprintf(os.Stderr, "Code is: %[1]s\nOn the other computer run\n\ncroc %[2]s%[1]s\n", c.Options.SharedSecret, flags.String())
+	fmt.Fprintf(os.Stderr, "Code is: %[1]s\nOn the other computer run\n\nCROC_SECRET=%[1]s croc %[2]s\n", c.Options.SharedSecret, flags.String())
```

Comment:
This isn't necessary on Windows. Can you change it to only print this on Unix systems (you can use `runtime.GOOS == "windows"`)

Reply:
The receiver system may not be the same as the sender. I changed this to print both commands.